preflight-checks/nccl-loopback/pkg/health/reporter_test.go

```
@@ -0,0 +1,208 @@
+
+	return captured[len(captured)-1]
+}
+
+// startTestConnector serves a capturingConnector on a Unix socket in a
+// temporary directory and returns the socket path.
+func startTestConnector(t *testing.T) (string, *capturingConnector) {
+	t.Helper()
+
+	socketPath := filepath.Join(t.TempDir(), "pc.sock")
+
+	lis, err := net.Listen("unix", socketPath)
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)